packages/core/src/__tests__/ai-brief-consolidated.test.ts

```diff
@@ -0,0 +1,80 @@
+import { getAiBriefContentFromBundle } from "../command/ai-brief/shared";
+import { makeBundle } from "../external/fhir/__tests__/bundle";
+import { stringToBase64 } from "../util/base64";
+
+describe("getAiBriefContentFromBundle", () => {
+  it("returns undefined when bundle has no Binary resource", () => {
+    const bundle = makeBundle({ entries: [] });
+    const result = getAiBriefContentFromBundle(bundle);
+    expect(result).toBeUndefined();
+  });
+
+  it("returns undefined when bundle has invalid Binary resource", () => {
+    const bundle = makeBundle({
+      entries: [
+        {
+          resourceType: "Binary",
+          id: "123",
+          // missing required fields
+        },
+      ],
+    });
+    const result = getAiBriefContentFromBundle(bundle);
+    expect(result).toBeUndefined();
+  });
+
+  it("returns undefined when Binary resource has no data", () => {
+    const bundle = makeBundle({
+      entries: [
+        {
+          resourceType: "Binary",
+          id: "123",
+          contentType: "text/plain",
+          // data field missing
+        },
```

Comment:
if we add `data` it still won't return it because it's missing `meta.source`, right? We should build a fully valid Binary with just the `data` field missing IMO.